html/config.html

```diff
@@ -0,0 +1,666 @@
+            <input type="form" id="rss_feed3_name" class="opt">
+            <input type="form" id="rss_feed3" class="extra">
+          </div>
+          <p></p>
+          <div class="var">
+            <input type="checkbox" id="rss_feed4_on">   Feed 4:
+            <input type="form" id="rss_feed4_name" class="opt">
+            <input type="form" id="rss_feed4" class="extra">
+          </div>
+          <p></p>
+          <div class="var">
+            <input type="checkbox" id="rss_feed5_on">   Feed 5:
+            <input type="form" id="rss_feed5_name" class="opt">
+            <input type="form" id="rss_feed5" class="extra">
+          </div>
+          <p></p>
+          <div class="var">
+            <input type="checkbox" id="rss_feed6_on">   Feed 6:
+            <input type="form" id="rss_feed6_name" class="opt">
+            <input type="form" id="rss_feed6" class="extra">
+          </div>
+          <p></p>
+          <div class="var">
+            <input type="checkbox" id="rss_feed7_on">   Feed 7:
+            <input type="form" id="rss_feed7_name" class="opt">
+            <input type="form" id="rss_feed7" class="extra">
+          </div>
+          <p></p>
+          <div class="var">
+            <input type="checkbox" id="rss_feed8_on">   Feed 8:
+            <input type="form" id="rss_feed8_name" class="opt">
+            <input type="form" id="rss_feed8" class="extra">
+          </div>
+          <p></p>
+          <div class="var">
+            <input type="checkbox" id="rss_feed9_on">   Feed 9:
+            <input type="form" id="rss_feed9_name" class="opt">
+            <input type="form" id="rss_feed9" class="extra">
+          </div>
+          <p></p>
+          <div class="var">
+            <input type="checkbox" id="rss_feed10_on">   Feed 10:
+            <input type="form" id="rss_feed10_name" class="opt">
+            <input type="form" id="rss_feed10" class="extra">
+          </div>
+          <p></p>
+
+        </div>
+        <p class="pad"></p>
+        <div class="settings">Log</div>
+        <p></p>
+        <div class="var">
+          <div class="tooltip">Log Level:
+            <span class="tooltiptext">Change this to 'DEBUG', if you need to troubleshoot issues.</span>
+          </div>
+          <select id="log_level" class="opt">
+            <option value="INFO">INFO</option>
+            <option value="DEBUG">DEBUG</option>
+          </select>
+        </div>
+        <p></p>
+        <div class="var">
+          <div class="tooltip"><a href="log">Go to Log ></a>
+            <span class="tooltiptext">Click this link to show the log.</span>
+          </div>
+        </div>
+-->
+      </div>
+      <br><br>
+      </form>
+      <div class="submit"><input type="submit" id="update" value="Update" class="submit" onclick="test_save()"></div>
+<!---"config_helper('save')"></div> --->
+      <br><br><br><br>
+    </div>
+  </body>
+</html>
```

Comment:
Please remove all the commented out code. It should be added in when the relevant bits are added. Personally, I would save the diff of the needed bits and the full version as a patch and just reapply it in a later.

Reply:
got it.